modules/liveIntentIdSystem.js

```diff
@@ -14,6 +14,7 @@ import { MinimalLiveConnect } from 'live-connect-js/esm/minimal-live-connect.js'
 
 const MODULE_NAME = 'liveIntentId';
 export const storage = getStorageManager({gvlid: null, moduleName: MODULE_NAME});
+const defaultRequestedAttributes = {'nonId': true}
```

Comment:
The requirements from https://docs.google.com/document/d/16rEBF_RkkA_0-C142sKC6gbLm3ucCtGSZyXQ1NQGEL0/edit and https://liveintent.atlassian.net/browse/NAVY-484 seems to be contradictory as of what should be the default for uid2. Let's take it to our sync with @deepthivenkat.